src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -279,7 +291,11 @@ public final void visitToken(DetailAST blockCommentNode) {
             }
 
             if (result.getParseErrorMessage() == null) {
-                processTree(result.getTree());
+                final DetailNode javadocTree = result.getTree();
+                if (doesCheckAcceptJavadocWithUnclosedHtmlTag()
+                        || !JavadocUtils.hasUnclosedTag(javadocTree)) {
```

Comment:
I sort-of dislike this as we are completely skipping javadoc if it has an unclosed tag and check doesn't accept it.
If this was completely unintentional from the user, they will never know there is a problem or what javadocs we are skipping. The amount of false positive reports to us will increase once we start using this more and more.

Should we instead print a violation on the javadoc saying it can't be processed because of an unclosed tag and maybe have a separate severity field or boolean field to turn that specific violation on/off?